docs/src/main/paradox/error.md

```diff
@@ -37,6 +40,9 @@ in configuration:
 The `Projection` will be restarted if it fails, for example if the offset can't be saved or processing of an
 envelope fails after applying the @ref:[Handler recovery](#handler-recovery).
 
+The following example is using the `CassandraProjection` but the same can be used with `JdbcProjection` or
+`SlickProjection`.
+
```

Comment:
Even if we don't plan on adding any other Implementation ATM, this will be a PITA to maintain. Maybe rewrite as:

```suggestion
The following example is using the `CassandraProjection` but the same can be used with any other [offset storage](overview.md).
```

Reply:
Technically, it's not about the offset storage, but about the functionality a projection implementation is offering.

Reply:
ok, but then it's on more places so I'll fix from my branch

Reply:
for flow, Future, and actor based handler it's about offset storage